server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestGetIndexTemplateAction.java

```diff
@@ -47,6 +49,10 @@
 
     private static final Set<String> RESPONSE_PARAMETERS = Collections.unmodifiableSet(Sets.union(
         Collections.singleton(INCLUDE_TYPE_NAME_PARAMETER), Settings.FORMAT_PARAMS));
+    private static final DeprecationLogger deprecationLogger = new DeprecationLogger(
+            LogManager.getLogger(RestGetIndexTemplateAction.class));
+    public static final String INCLUDE_TYPE_NAME_DEPRECATION_MESSAGE = "[types removal]" +
```

Comment:
I think we should call this `TYPES_DEPRECATION_MESSAGE` for consistency with the other APIs (including put mapping).